src/annotator/test/bucket-bar-test.js

```diff
@@ -80,30 +80,37 @@ describe('BucketBar', () => {
       assert.calledWith(fakeAnnotator.selectAnnotations, fakeAnnotations, true);
     });
 
-    context('when scrollables provided', () => {
-      let scrollableEls = [];
+    context('when `contentContainer` is specified', () => {
+      let container;
 
       beforeEach(() => {
-        const scrollableEls1 = document.createElement('div');
-        scrollableEls1.className = 'scrollable-1';
-        const scrollableEls2 = document.createElement('div');
-        scrollableEls2.className = 'scrollable-2';
-        scrollableEls.push(scrollableEls1, scrollableEls2);
-        document.body.appendChild(scrollableEls1);
-        document.body.appendChild(scrollableEls2);
+        container = document.createElement('div');
+        container.className = 'scrollable-1';
```

Comment:
It is not used, so maybe it could be deleted.

Reply:
Indeed. I'll remove this.